File: bitget-java-sdk-api/src/main/java/com/bitget/openapi/api/v3/UtaCopyApi.java
package com.bitget.openapi.api.v3;

import com.bitget.openapi.dto.response.ResponseResult;
import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.GET;
import retrofit2.http.POST;
import retrofit2.http.QueryMap;

import java.util.Map;

public interface UtaCopyApi {

    // -------- trader info --------

    @GET("/api/v3/copy/futures/trading-pairs")
    Call<ResponseResult> tradingPairs();

    @GET("/api/v3/copy/futures/position-summary")
    Call<ResponseResult> positionSummary();

    // -------- copy trading transfer --------

    @GET("/api/v3/copy/futures/max-transferable")
    Call<ResponseResult> maxTransferable(@QueryMap Map<String, String> paramMap);

    @POST("/api/v3/copy/futures/transfer")
    Call<ResponseResult> transfer(@Body Map<String, Object> paramMap);

    @GET("/api/v3/copy/futures/transfer-record")
    Call<ResponseResult> transferRecord(@QueryMap Map<String, String> paramMap);
}

File: bitget-node-sdk-api/src/lib/v3/UtaCopyApi.ts
import {BaseApi} from '../BaseApi';

// Covers copy trading transfers and trace queries.
// Merged from: CopyTradingTransferController, UtaTraceController
export class UtaCopyApi extends BaseApi {

    // -------- copy futures transfer --------

    // GET /api/v3/copy/futures/max-transferable
    maxTransferable(qsOrBody: object) {
        const url = '/api/v3/copy/futures/max-transferable';
        const headers = this.signer('GET', url, qsOrBody)
        return this.axiosInstance.get(url, {headers, params: qsOrBody})
    }

    // POST /api/v3/copy/futures/transfer
    transfer(qsOrBody: object) {
        const url = '/api/v3/copy/futures/transfer';
        const headers = this.signer('POST', url, qsOrBody)
        return this.axiosInstance.post(url, qsOrBody, {headers})
    }

    // GET /api/v3/copy/futures/transfer-record
    transferRecord(qsOrBody: object) {
        const url = '/api/v3/copy/futures/transfer-record';
        const headers = this.signer('GET', url, qsOrBody)
        return this.axiosInstance.get(url, {headers, params: qsOrBody})
    }

    // -------- trace (follower / trader query) --------

    // GET /api/v3/copy/futures/trading-pairs
    futuresTradingPairs(qsOrBody: object) {
        const url = '/api/v3/copy/futures/trading-pairs';
        const headers = this.signer('GET', url, qsOrBody)
        return this.axiosInstance.get(url, {headers, params: qsOrBody})
    }

    // GET /api/v3/copy/futures/position-summary
    futuresPositionSummary(qsOrBody: object) {
        const url = '/api/v3/copy/futures/position-summary';
        const headers = this.signer('GET', url, qsOrBody)
        return this.axiosInstance.get(url, {headers, params: qsOrBody})
    }
}
